gulpfile.js

```diff
@@ -114,3 +117,42 @@ gulp.task("docs:check", function() {
     });
 });
 // #endregion
+
+
+gulp.task("resolve-vars", function() {
+    let themes = glob.sync(paths.sass.themes);
+    const { parse } = require('sass-variable-parser');
+    const cwd = process.cwd();
+
+    themes.forEach( theme => {
+
```

Comment:
What JS guidelines do we use for this repo? In Kendo for jQuery, we do not add an empty line at the beginning and the end of a function, is this okay for this repo?

Reply:
we'll have to think on that.